src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: #000;
+  --hover-color: #00acdc;
+  --header-background-color: #fff;
+}
+
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to include fallback fonts in the font-family as a backup in case the primary font isn’t available. For example, 'font-family: Roboto, Arial, sans-serif;'